pkg/scale/decode_test.go

```diff
@@ -132,6 +132,121 @@ func Test_decodeState_decodeSlice(t *testing.T) {
 	}
 }
 
+// // Rust code to encode a map of string to struct.
+// let mut btree_map: BTreeMap<String, User> = BTreeMap::new();
```

Comment:
This is Rust code used to get expected encoded data.

Reply:
@axaysagathiya I think it is not needed to put this comment block here, could you provide a link to the actual Rust code that does the encode?

Reply:
@EclesioMeloJunior this Rust code is written by me to get scale encoded data. Do you want me to provide a Gist link ?

Reply:
Not really, I mean your comment starts with: 

```
// // Rust code to encode a map of string to struct.
```

But actually, there is no code that encodes a map in your comment block, you're just adding two entries into a `BTreeMap`, that is why I would suggest removing this comment

Reply:
@kishansagathiya cool, I thought it was taken from somewhere else so I am fine to leave it as is. Want to avoid situations where we don't know where bytes came from.

Reply:
Oh, okay I did understand that you were trying to explain how rust encodes but you're explaining how you generated the encoded bytes that you're using in the test (my bad 😅)

Reply:
> Oh, okay I did understand that you were trying to explain how rust encodes but you're explaining how you generated the encoded bytes that you're using in the test (my bad sweat_smile)

:laughing: